.gitignore

```diff
@@ -6,3 +6,4 @@
 /tests/acceptance.conf.php
 /tests/smoke.conf.php
 /vendor/
+/.idea/
```

Comment:
It is the config folder of IDE phpstorm avoids wrong commit. Don't know how often it is used...